src/SimpleVersion.Core/Pipeline/VersionContext.cs

```diff
@@ -35,13 +35,15 @@ public VersionContext(Git.IRepository repository)
         public Git.IRepository Repository { get; }
 
         private VersionResult SetIntialResult()
-        {
+        {
+            var sha = Repository.Head.Tip?.Sha;
             return new VersionResult
             {
                 BranchName = Repository.Head.FriendlyName,
                 CanonicalBranchName = Repository.Head.CanonicalName,
-                Sha = Repository.Head.Tip?.Sha
-            };
+                Sha = sha,
+                Sha7 = (string.IsNullOrEmpty(sha) || sha.Length < 8) ? null : sha.Substring(0, 7)

```

Comment:
@asci13 Should this be `sha.length < 7`? If the sha is 7 or greater you should still set the `sha7`

Reply:
@Kieranties yup, you're right - adjusted accordingly